hack/ovirt-populator/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.19.3-alpine3.16 as builder
+WORKDIR /app
+
+COPY . ./
+WORKDIR /app/cmd/ovirt-populator
+
+# When debugging by replacing the lib-volume-populator with a local modified copy
+# COPY . ./
+
+RUN CGO_ENABLED=0 go build -o /ovirt-populator
+
+FROM quay.io/centos/centos:stream9
```

Comment:
and I see there's stream9-minimal in [quay.io/centos/centos](https://quay.io/repository/centos/centos?tab=tags&tag=latest) - how about trying it?